lib/ecto/query.ex

```diff
@@ -439,18 +439,18 @@ defmodule Ecto.Query do
 
   ## `where`, `having` and a `join`'s `on'
 
-  `dynamic` can be interpolated at the root of a `where`, `having` or
+  `conditions` can be interpolated at the root of a `where`, `having` or
```

Comment:
I thin in this case it should be `dynamic` :) Or say "The result of the `dynamic` macro" or similar.